modules/zero-trust-manager-create-route-oidc.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * security/zero_trust_workload_identity_manageer/zero-trust-manager-oidc-federation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="zero-trust-manager-create-route-oidc_{context}"]
+= Creating a route for OIDC
+
+The Zero Trust Workload Identity Manager controller can automatically create OIDC routes that enable external access to the SPIRE OIDC Discovery Provider endpoints through OpenShift Routes. When a `SpireOIDCDiscoveryProvider` is deployed, the enhanced controller automatically creates routes to provide secure, externally accessible OIDC endpoints for workload identity verification and JWT token validation in multi-cluster and hybrid cloud environments.
+
+.Prerequisites
+
+* Either the Vault or Entra ID OIDC Provider is installed.
+
+* An externally reachable hostname that is used for the route.
+
+* Install a Transport Layer Socket (TLS) and its corresponding private key if creating an HTTPS route.
```

Comment:
Use the "Prerequisites" documented in "Configure external certificate for the managed OIDC discovery provider route"